.ci_support/environment-openmpi-mac.yml

```diff
@@ -0,0 +1,11 @@
+channels:
+- conda-forge
+dependencies:
+- coverage
+- lammps =2024.06.27=*_mpi_openmpi_*
```

Comment:
Related to the LAMMPS bug https://github.com/conda-forge/lammps-feedstock/issues/207